articles/overview/what-is-qsharp.md

```diff
@@ -3,26 +3,24 @@ title: What is Q#?
 description: Learn about the Q#, a programming language created by Microsoft to develop applications for quantum computers
 author: natke
 ms.author:  nakersha
-ms.date: 10/16/2019
+ms.date: 10/22/2019
 ms.topic: article
 uid: microsoft.quantum.overview.qsharp
 ---
 
 # What is Q#?
 
-Q# is a programming language created by Microsoft to develop applications for quantum computers.
+Q# is a programming language with features that are special to quantum computing. Q# is great for writing quantum programs. Q# provides quantum programmers a framework that allows you to focus on the algorithms without having to care about technical details like gate sequence optimization or the physical implementation of a quantum computer.
 
-The Q# programming language takes care of all the quantum computer's internal logic and provides you with an intuitive set of types, operations and logic expressions to develop algorithms without having to worry about the internal logic of the quantum computer.
+The Q# programming language provides you with an intuitive set of types, operations and logic expressions to develop algorithms without having to worry about the internal logic of the quantum computer.
 
 ## We want to code algorithms, not draw circuits
```

Comment:
"draw circuits" threw me here.
How about: Code quantum algorithms and apps; skip drawing circuits
Did understand this?

Reply:
Changed to Code Algorithms